Core/ReadInput/InputReader.cs

```diff
@@ -94,9 +101,10 @@ public SemanticVersion ReadVersionChoice (string prompt, IReadOnlyCollection<Sem
   /// <param name="possibleAnswers">Must never just be integers between 0 and the number of possible answers</param>
   public string ReadStringChoice (string prompt, IReadOnlyCollection<string> possibleAnswers)
   {
-    if (_console.Profile.Capabilities.Interactive)
+    if (_console.Profile.Capabilities.Ansi)
```

Comment:
Given your explaination, we should move this change into its own Ticket/PR. It has nothing to do with the color scheme.